lib/foreman/renderer.rb

```diff
@@ -99,7 +113,10 @@ def snippet(name, options = {})
         begin
           return unattended_render(template, nil, options[:variables] || {})
         rescue => exc
-          raise "The snippet '#{name}' threw an error: #{exc}"
+          # fixes the backtrace but ideally should raise better error message with Foreman rendering exception, same everywhere in this file
```

Comment:
Hm, the WrappedException hierarchy does not work in here, I want all exceptions raised by renderer have one commont parent so it's easy to rescue them only. Not every renderer exception will wrap another exception, WrappedException should have been concern probably. Anyway I'll keep exceptions refactoring for later, I removed the comment.